tests/Fixer/Casing/ConstantCaseFixerTest.php

```diff
@@ -164,8 +159,21 @@ class Foo
             ['<?php Null/**/::test();'],
             ['<?php True//
                                 ::test();'],
-            ['<?php trait False {}'],
-            [
+            ['<?php class Foo { public function Bar() { $this->False = 1; $this->True = 2; $this->Null = 3; } }'],
+        ];
+
+        foreach ($tests as $index => $test) {
+            yield $index => $test;
+        }
+
+        if (\PHP_VERSION_ID < 70000) {
```

Comment:
Should probs make a separate test and explicitly skip it.

Reply:
:+1:  thanks, done!